src/components/ModalWrapper/ModalWrapper-story.js

```diff
@@ -106,8 +110,7 @@ storiesOf('ModalWrapper', module)
         <TextInput
           id="test2"
           placeholder="Hint text here"
-          label="Text Input:"
-          data-modal-primary-focus
```

Comment:
Just wanted to double-check if removal of `data-modal-primary-focus` here is intentional.

Reply:
I removed it to keep it consistent with the other storybook examples with the `selectorPrimaryFocus` prop. I believe none of the other examples contain this data attribute so it defaults to the focusing on primary button. If the user wants to test it autofocusing on a different element they can use the Storybook knob to choose a different selector